[Core]/Internal/CecilExtensions.cs

```diff
@@ -119,23 +119,32 @@ internal static class CecilExtensions {
         }
 
         private static void CorrectBranchSizes(ILProcessor il) {
-            var offset = 0;
-            foreach (var instruction in il.Body.Instructions) {
-                offset += instruction.GetSize();
-                instruction.Offset = offset;
-            }
 
-            foreach (var instruction in il.Body.Instructions) {
-                var opCode = instruction.OpCode;
-                if (opCode.OperandType != OperandType.ShortInlineBrTarget)
-                    continue;
+            bool madeAChange;
 
-                var operandValue = ((Instruction)instruction.Operand).Offset - (instruction.Offset + instruction.GetSize());
-                if (operandValue >= sbyte.MinValue && operandValue <= sbyte.MaxValue)
-                    continue;
+            // expanding one short branch can force a previous short branch to be expanded, so we have to do this repeatedly

```

Comment:
Does this mean that instead of doing that repeatedly we can go once but in reverse?